admin_guide/cluster_management.rst

```diff
@@ -77,6 +77,22 @@ To add a new node to the cluster, you first need to add the DNS name or IP addre
 
 The new node is available for shards of new distributed tables. Existing shards will stay where they are unless redistributed, so adding a new worker may not help performance without further steps.
 
+.. note::
+
+   As of Citus 8.1, workers use encrypted communication by default. A new node running version 8.1 or greater will refuse to talk with other workers who do not have SSL enabled. When adding a node to an unencrypted cluster, you must reconfigure the new node.
```

Comment:
Maybe "unencrypted cluster" is sloppy terminology, but I wanted to keep it short.

Reply:
yeah, first time I read over this, but now it started nagging that unencrypted cluster is easily interpreted as encryption at rest. How about;

`When adding a node to a cluster without encrypted communication, you must reconfigure the new node before creating the citus extension.`

The last part I added is an important part that is missing currently.